identity/resource_group.go

```diff
@@ -136,6 +154,10 @@ func ResourceGroup() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 			},
+			"allow_workspace_access": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
```

Comment:
Should we support existing behaviour when users get workspace access by default? Otherwise they will lose it as we don't specify default for new field.

Reply:
hmm its an optional field we can also make it computed and let the server take care of it? but ya these fields dont exist on workspaces without sqlA enabled.